Automattic/jetpack
# Social Notes: Allow multiple paragraphs

Fixes https://github.com/Automattic/jetpack-reach/issues/232

## Proposed changes:

* Removed the template lock
* Added a filter that allow only core/paragraph blocks to be added

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/232
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* With this PR create a new Social Note.
* You should be able to add multiple paragraphs to it
